allcontainers.js

```diff
@@ -98,7 +105,17 @@ function allContainers (opts) {
         return result.emit('error', err)
       }
 
-      containers.forEach(emit)
+      var activeContainers = [];
+      function findPausedContainers(container) {
+        if (container.Status.indexOf("Paused") !== -1) {
+          pausedPreheatContainers[container.Id] = container;
+        } else {
+          activeContainers.push(container);
+        }
+      }
```

Comment:
let's not worry too much about it.. this would probably need some perf refactor (start and stop should not be initialized there and moved somewhere else).